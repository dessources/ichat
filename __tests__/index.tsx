import { render } from "@testing-library/react";
import Home from "@/pages/index";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  // code to make test fail on purpose
  //   expect(4).toBe(5);
  expect(container).toMatchSnapshot();
});
