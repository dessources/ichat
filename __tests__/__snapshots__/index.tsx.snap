// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  aria-hidden="true"
>
  <main
    class="main"
  >
    <div
      class="header"
    >
      <h1>
        Ichat
      </h1>
    </div>
  </main>
</div>
`;
